install/actualbudget-install.sh

```diff
@@ -45,6 +46,7 @@ ACTUAL_UPLOAD_DIR=/opt/actualbudget/server-files
 PORT=5006
 EOF
 cd /opt/actualbudget
+$STD git checkout "$RELEASE"
```

Comment:
Instead of using git it would probably be better to pull the tag tarball from `https://api.github.com/repos/actualbudget/actual-server/tarball/refs/tags/${RELEASE_SERVER}` like in https://github.com/community-scripts/ProxmoxVE/blob/d3d6ec8b209a112c002935ca4fbee075553fdef5/install/apache-guacamole-install.sh#L57

Reply:
That was my initial approach but I realized using git made the changes much simpler. Since other scripts use tarballs though I'll do the same here.